app/src/androidTest/java/com/github/h3lp3rs/h3lp/NotificationTest.kt

```diff
@@ -0,0 +1,75 @@
+package com.github.h3lp3rs.h3lp
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.*
+import com.github.h3lp3rs.h3lp.notification.NotificationService
+import junit.framework.Assert.assertEquals
+import org.junit.Rule
+import org.junit.Test
+
+/**
+ * Should use uiDevice instead of espresso to be able to test element outside the scope of application
+ */
+
+class NotificationTest {
+
+    private val ctx: Context = ApplicationProvider.getApplicationContext()
+    private val uiDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+    private val TITLE : String = "Title"
+    private val DESCRIPTION : String = "Description"
+
+    @get:Rule
+    val testRule = ActivityScenarioRule(
+        MainPageActivity::class.java
+    )
+
+
+    @Test
+    fun sendSimpleNotificationWork() {
+        NotificationService.createNotificationChannel(ctx)
```

Comment:
Could be in a startup method (annoted with @Begin) since it's used in both tests though this would not be necessary if you implement Emilien's comments regarding the fact that sending a notification should already create the notification channel if it wasn't already created.